public/docs/ts/latest/guide/animations.jade

```diff
@@ -337,3 +338,19 @@ figure
 
 :marked
   One group animates the element transform and width. The other animates the opacity.
+
+:marked
+  ## Animation callbacks
+
+  A callback is fired when an animation is started and also when it is done.
+
+  In the keyframes example, we have a `trigger` called `@flyInOut`. There we can hook
+  those callbacks like:
+
++makeExample('animations/ts/app/hero-list-multistep.component.ts', 'template')(format=".")
+
+:marked
+  The callbacks receives an `AnimationTransitionEvent` which contain useful properties like `fromState`,
```

Comment:
The callbacks receive an `AnimationTransitionEvent` which contains useful properties such as `fromState`,